### PR TITLE
Math magicians: Components

### PR DESCRIPTION
**Pull request by IjayAbby**
### Project requirements.

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components.
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @IjayAbby  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

I wish I could use inline comments but the changes ain't on this PR.

1. According to the project requirement, we need to make use of a class component.

Our `Calculator.js` and `App.js` should be a class component, example below;

```
import React, { Component } from 'react';

class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}
```

2. **Kindly update your `README` file before merging to `dev`**
**Let's make your repo look more professional by updating your `README` file**.
Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
@IjayAbby 

Quick one: if you get any linter error while converting the functional component to class component, try to use `extends PureComponent` instead of just `extends Component ` as the calculator component has no state for now.

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import './App.css';
-import Calc from './components/Calc';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (bellom) [inline src/App.js]: @IjayAbby Good job changing the Calculator function to class-based but for some reason, you didn't change `App` to the class-based component as I mentioned in the previous review, kindly do that.

Example:

`class App extends Component {`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @IjayAbby  👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @IjayAbby  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**